etc/calico/confd/templates/bird_ipam.cfg.template

```diff
@@ -34,7 +34,10 @@ filter calico_kernel_programming {
 {{- end}}
 {{range ls "/v1/ipam/v4/pool"}}{{$data := json (getv (printf "/v1/ipam/v4/pool/%s" .))}}
   if ( net ~ {{$data.cidr}} ) then {
-{{if $data.ipip_mode}}{{if eq $data.ipip_mode "cross-subnet"}}
+{{- if $data.vxlan_mode}}
```

Comment:
Thought we default to `Never` on read?  That might make this condition true when it shouldn't be.

Reply:
In the backend v1 code, "Never" is represented as an empty string :/

Also, I'd expect existing tests to be failing otherwise.